web-transport-quinn/src/client.rs

```diff
@@ -91,9 +97,17 @@ impl ClientBuilder {
         certs: Vec<CertificateDer>,
     ) -> Result<Client, ClientError> {
         let hashes = certs.iter().map(|cert| {
-            aws_lc_rs::digest::digest(&aws_lc_rs::digest::SHA256, cert)
+            #[cfg(feature = "aws-lc-rs")]
+            let digest = aws_lc_rs::digest::digest(&aws_lc_rs::digest::SHA256, cert)
+                .as_ref()
+                .to_vec();
+
+            #[cfg(feature = "ring")]
```

Comment:
This will cause warnings if both the ring and aws-lc-rs feature are enabled. It also would cause ring to be preferred.